core/whistle_couch-1.0.0/src/couch_util.erl

```diff
@@ -1021,6 +1034,23 @@ publish_doc(#db{name=DbName}, Doc, JObj) ->
             end
     end.
 
+-spec publish_db(couchbeam_db(), atom()) -> 'ok'.
+publish_db(Db, Action) ->
+    Type = 'database',
+    Props =
+        [{<<"Type">>, 'database'}
+         ,{<<"ID">>, Db}
+         ,{<<"Rev">>, <<"0">>}
```

Comment:
I would say, let's move Rev to optional and not include it here, or create a `publish_db_update` to differentiate it from doc updates.